google/cloud/spanner_v1/batch.py

```diff
@@ -183,11 +189,17 @@ def commit(self, return_commit_stats=False, request_options=None):
         # Request tags are not supported for commit requests.
         request_options.request_tag = None
 
+        max_commit_delay = None
+        if max_commit_delay_ms:
+            max_commit_delay = Duration()
+            max_commit_delay.FromMilliseconds(max_commit_delay_ms)
```

Comment:
We can avoid this handwritten logic by getting input `max_commit_delay_ms` as [datetime.timedelta](https://docs.python.org/3/library/datetime.html#timedelta-objects). This way we can directly pass the input to the `CommitRequest` without any additional logic.

Let me know if you have better approach?

Reply:
done.